homeassistant/components/zwave_js/__init__.py

```diff
@@ -134,6 +148,31 @@ def async_on_node_removed(node: ZwaveNode) -> None:
         # note: removal of entity registry is handled by core
         dev_reg.async_remove_device(device.id)
 
+    @callback
+    def async_on_value_notification(notification: ValueNotification) -> None:
+        """Relay stateless value notification events from Z-Wave nodes to hass."""
+        device = dev_reg.async_get_device({get_device_id(client, notification.node)})
+        value = notification.value
+        if notification.metadata.states:
+            value = notification.metadata.states.get(str(value), value)
+        hass.bus.async_fire(
+            ZWAVE_JS_EVENT,
+            {
+                ATTR_TYPE: "value_notification",
+                CONF_DOMAIN: DOMAIN,
```

Comment:
We only use `ATTR_` constants for config keys. We should make an `ATTR_DOMAIN` constant.

Reply:
fixed